package.json

```diff
@@ -11,6 +11,7 @@
 		"@typescript-eslint/parser": "4.21.0",
 		"eslint": "7.23.0",
 		"eslint-config-prettier": "8.1.0",
+		"eslint-plugin-only-error": "^1.0.2",
```

Comment:
sorry @ZeroX-DG , there's already https://eslint.org/docs/user-guide/command-line-interface#-max-warnings option so please use that and avoid having a new dependency :D tks @ng-hai for the suggestion

Reply:
@thien-do yeah I already tried that. It will make eslint exit with the error status but the output still show as warning. Is that what we want?

**Max warning output:**
![image](https://user-images.githubusercontent.com/12984316/114287936-5e82e600-9abf-11eb-8816-b58acf7cfe73.png)

**This implmentation output:**
![image](https://user-images.githubusercontent.com/12984316/114287958-82dec280-9abf-11eb-8e68-e4e8c3a4c183.png)

Reply:
Yes that's expected. It's ok to have the log says "warning". What we want is simply for the check is failed so the PR is blocked from merging.

Reply:
Using max-warnings works as expected
<img width="1046" alt="Screen Shot 2021-04-11 at 14 16 02" src="https://user-images.githubusercontent.com/8168339/114295609-7f076c00-9ad0-11eb-9915-3e34ac561351.png">
<img width="1039" alt="Screen Shot 2021-04-11 at 14 10 18" src="https://user-images.githubusercontent.com/8168339/114295603-7b73e500-9ad0-11eb-88c4-974881a4201b.png">
<img width="1036" alt="Screen Shot 2021-04-11 at 14 10 35" src="https://user-images.githubusercontent.com/8168339/114295608-7e6ed580-9ad0-11eb-94cf-6fccb0c4ce0b.png">

Reply:
Wow tks @lqt93 that looks like what I want. I'm surprised that GitHub can show the line